charts/steward/values.yaml

```
@@ -42,16 +44,18 @@ tenantController:
       cpu: 10m
   podSecurityContext: {}
     # fsGroup: 2000
-  securityContext: {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  securityContext:
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1000
+    runAsGroup: 1000
   nodeSelector: {}
   affinity: {}
   tolerations: []
+  mappableClusterRoles: ["steward-tenant"]
```

'mappable' seems to be the wrong term, at least it's not intuitive.

The annotations on client namespace is called `tenant-role`, so I propose:
```suggestion
  tenantRoles: ["steward-tenant"]
```